### ISSUE TITLE
Superseded Selectors API

### ISSUE THREAD

**Issue by dontcallmedom** [w3c/webappswg#21]
[Selectors API](https://www.w3.org/TR/selectors-api/) was published as a Rec in 2013, but its content has since been merged in [WHATWG DOM](https://dom.spec.whatwg.org/).

Unfortunately, this was not listed in the [W3C-WHATWG MoU](https://www.w3.org/2019/04/WHATWG-W3C-MOU.html#transition), so we should explicitly get Director's approval to obsolete it and ask for it to be handled the same way other WHATWG-migrated specs get treated.

**Comment by marcoscaceres** [comment]
@dontcallmedom, do we need to go through the TAG deprecation process?

**Comment by dontcallmedom** [comment]
No, since WebApps is (or can be argued to be) the group who produced that spec, it can [initiate the request to the Director](https://www.w3.org/2019/Process-20190301/#rec-rescind):
>The process of obsoleting, rescinding, or restoring a Recommendation can be initiated [...] from any of the following:
>*  The Working Group who produced, or is chartered to maintain, the Recommendation.

**Comment by marcoscaceres** [comment]
Ok, cool. We can do that! :)

**Comment by marcoscaceres** [comment]
@siusin is on it :) Could you let us know if we need to do anything (e.g., CFC) or whatever.

**Comment by LJWatson** [comment]
We need to put a CFC to the WG then request the transition. 

Looking at the [Process](https://www.w3.org/2019/Process-20190301/#rec-rescind), we should probably opt for superseded rather than obsolete, since the API will be superseded by what's in the DOM spec.

The DOM LS/review draft is expected to transition to CR by the end of this month, so we should wait until that happens before putting the CFC out.

**Comment by marcoscaceres** [comment]
[CFC sent](https://lists.w3.org/Archives/Public/public-webapps/2020JanMar/0001.html). Issue is blocked until the end of the month awaiting working group feedback. However, we can probably make any updates to the document in parallel. According to the process document, "only the name and explanation change" - so pretty plainless.

**Comment by marcoscaceres** [comment]
The CFC has passed. No objections! 🎉

**Comment by siusin** [comment]
[Level 2](https://www.w3.org/TR/selectors-api2/) has been retired on 16-04-2020 (a note was added to the SoTD).
Level 1 is pending on [AC review.](https://lists.w3.org/Archives/Member/w3c-ac-members/2020AprJun/0016.html)

**Comment by marcoscaceres** [comment]
Appreciate the update @siusin. Thank you.

**Comment by marcoscaceres** [comment]
Assuming no complaints, let's close this as done.
